examples/gallery/3d_plots/scatter3d.py

```diff
@@ -36,11 +47,16 @@
 
 # Define a colormap to be used for three categories, define the range of the
 # new discrete CPT using series=(lowest_value, highest_value, interval), use
-# color_model="+cSetosa,Versicolor,Virginica" to write the discrete color
+# color_model="+csetosa,versicolor,virginica" to write the discrete color
 # palette "cubhelix" in categorical format and add the species names as
 # annotations for the colorbar
 pygmt.makecpt(
-    cmap="cubhelix", color_model="+cSetosa,Versicolor,Virginica", series=(0, 2, 1)
+    cmap="cubhelix",
+    # Use the minum and maximum of the categorical number code
```

Comment:
Pandas assigns numbers (in an alphabetical manner) to the individual categories, when converting `df.species` to the categorical data type:
```
df.species = df.species.astype(dtype="category")
```
 - [Color points by categories](https://www.pygmt.org/dev/gallery/symbols/points_categorical.html#sphx-glr-gallery-symbols-points-categorical-py): Adelie -> 0, Chinstrap -> 1, Gentoo -> 2
 - [3D Scatter plots](https://www.pygmt.org/dev/gallery/3d_plots/scatter3d.html#sphx-glr-gallery-3d-plots-scatter3d-py): setosa -> 0, versicolor -> 1, virginica -> 2

These numbers are used to define the range of the colormap (`series` of `pygmt.makecpt`):
```
    series=(df.species.cat.codes.min(), df.species.cat.codes.max(), 1),
```
Thus, I think we should use here 'categorical number code'. Also because this term is later used in the comment for the `color` parameter of the `pygmt.Figure.plot()` or `pygmt.Figure.3Dplot()` method:
```
    # Points colored by categorical number code
    color=df.species.cat.codes.astype(int),
```

Reply:
I'm not that familiar with panda's categorical data type, but looking at the documentation and the functions calls `df.species.cat.categories` and `df.species.cat.codes`, I think `categories` and `number code` are better.

Reply:
Hmm you're right, let's go with `categories` and `number code` then. Wasn't aware that pandas actually uses numbers for categorical data!